### PR TITLE
Return a template for canvas pages instead of a raw Response

### PR DESCRIPTION
**Pull request by marcospri**
Pretty basic HTML page with just the title and the body.

## Testing

- Launch https://hypothesis.instructure.com/courses/125/assignments/5142

It looks similar to the vession in `main` but it does include the title now.

### CODE REVIEW

File: lms/templates/api/canvas/page.html.jinja2 [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang=en>
+  <head>
+    <meta charset=utf-8>
+    <title>{{title}}</title>
+  </head>
+  <body>
+    <h1>{{title}}</h1>
```
Comment (marcospri) [inline lms/templates/api/canvas/page.html.jinja2:8]: This doesn't improve the actual output that much but later changes would be more obvious with the template in place.

**Comment by acelaya** [comment]
@marcospri can you rebase this branch? It seems to be showing more changes than it is actually introducing

**Comment by marcospri** [comment]
> @marcospri can you rebase this branch? It seems to be showing more changes than it is actually introducing

Done @acelaya , I merged the base PR and forgot about this one.

**Review by acelaya — APPROVED** [review]
👍🏼

**Pull request merged**